spec/facility_spec.rb

```diff
@@ -14,13 +29,92 @@
     end
   end
 
-  describe '#add service' do
+
+
+  describe 'add_service' do
     it 'can add available services' do
       expect(@facility.services).to eq([])
       @facility.add_service('New Drivers License')
       @facility.add_service('Renew Drivers License')
       @facility.add_service('Vehicle Registration')
-      expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+      @facility.add_service('Written Test')
+      @facility.add_service ('Renew License')
+      expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration','Written Test','Renew License'])
+    end
+  end
+
+
+
+  describe 'register vehicle' do
+    it 'registers a vehicle' do
+      @facility.register_vehicle(@cruz)
+      expect(@facility.registered_vehicles).to include(@cruz)
```

Comment:
Instead of checking that the return value includes @cruz, we want to be more explicit. With this setup, there should be exactly one Vehicle in this array, so it should read `expect(@facility.registered_vehicles).to eq([@cruz])` The more explicit our tests, the better.